magento/pwa-studio
# Update prettier config

## Description

Added the following parameters under the prettier.config.js file for tab width and remove trailing comma
```
    tabWidth: 4,
    trailingComma: 'none'
```

## Related Issue





Closes #2899.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. In packages/venia-concept directory
2. Run `yarn run prettier:fix your-custom-extension-directory` command
3. Check file update history
4. It will not add a trailing comma and result will be like
```
    const { data, error, loading } = useQuery(GET_BLOG_POST, {
        variables: { id: id }
    });
```

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
